crates/ide-assists/src/handlers/inline_call.rs

```diff
@@ -470,7 +471,7 @@ fn inline(
         }
     } else if let Some(stmt_list) = body.stmt_list() {
         ted::insert_all(
-            ted::Position::after(stmt_list.l_curly_token().unwrap()),
+            ted::Position::after(stmt_list.l_curly_token()?),
```

Comment:
I think we can just do the following
```suggestion
            ted::Position::after(stmt_list),
```

Reply:
Using `stmt_list.syntax()` here causes some tests to fail. So I go with `expect` here.